### DISCUSSION TITLE
trouble with opening up website from brackets

### DISCUSSION THREAD

**Discussion by yummyrice1234** [brackets-cont/brackets#320] [Q&A]
when I click the lighting bolt, It says that I need to relaunch the website. So I keep relaunching the website, but I ran into the same issue again and again. What am I doing wrong?

**Comment by abose** [comment]
Hi, thanks for reporting the issue. There has been some issues with live preview due to chrome updates. 

Please try upgrading to https://phcode.io , the next generation of Brackets that is built on a new live preview framework. That should most likely fix the problem.
